<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gunCamWgt</class>
 <widget class="QWidget" name="gunCamWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gunCamWgt</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#gunCamWgt{
background:transparent;
}
#titleWidget{
background:#ffffff;
border-top-left-radius:4px;
border-top-right-radius:4px;
}
#midWidget{
background:#eeeeee;
}
#bottomWidget{
background:#eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}
QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#333333;
}
#confirmBtn{
background:#ffd500;
border-radius:2px;
color:#333333;
}
#confirmBtn:pressed{
margin:1px;
}
#cancelBtn{
background:#ffffff;
border-radius:2px;
color:#333333;
}
#cancelBtn:pressed{
margin:1px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>球机192.168.2.110</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="closeBtn">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#closeBtn{
 border-image:url(:/image/image/popupx2.png);
}
#closeBtn:hover{
 border-image:url(:/image/image/popupx2b.png);
}
#closeBtn:pressed{
margin:1px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="scaleBtn">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#scaleBtn{
 border-image:url(:/image/image/windowbigerb.png);
}
#scaleBtn:pressed{
margin:1px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="midWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>666666</width>
          <height>666666</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
